source/includes/system.md

```diff
@@ -151,6 +151,25 @@ system.experiences.edit(a.id);
 
 Experiences may be retrieved in a variety of ways.
 
+## Login
+
+> Check if the user is logged in.
+```javascript
```

Comment:
Have you tried rendering this? _Almost_ positive a new line is required between the > and back ticks. Also: this is what I was referring to in the other comment about pulling comments into > remarks.

Reply:
You're right - forgot to add the tweak to the commit before pushing. Will update!